CHANGELOG.md

```diff
@@ -1,17 +1,16 @@
 # Release Notes
 
-## 10.1.5
-
-* Bump iOS to 10.2.0
-
 ## 10.1.4
 
+* Support for react native 0.74.x
```

Comment:
Are there any migration steps that will be required due to the change in architecture?